cmd/project/project_test.go

```diff
@@ -0,0 +1,206 @@
+package project
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/flagship-io/flagship/models"
+	"github.com/flagship-io/flagship/utils"
+	"github.com/flagship-io/flagship/utils/config"
+	"github.com/jarcoal/httpmock"
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestProjectCommand(t *testing.T) {
+	output, _ := utils.ExecuteCommand(ProjectCmd)
+	assert.Equal(t, output, "Manage your projects\n\nUsage:\n  project [create|edit|get|list|delete|toggle] [flags]\n  project [command]\n\nAvailable Commands:\n  completion  Generate the autocompletion script for the specified shell\n  create      Create a project\n  delete      Delete a project\n  edit        Edit a project\n  get         Get a project\n  help        Help about any command\n  list        List all projects\n  toggle      Toggle a project\n\nFlags:\n  -h, --help   help for project\n\nUse \"project [command] --help\" for more information about a command.\n")
+}
+
+func TestHelpCommand(t *testing.T) {
+	output, _ := utils.ExecuteCommand(ProjectCmd, "--help")
+	assert.Equal(t, output, "Manage your projects\n\nUsage:\n  project [create|edit|get|list|delete|toggle] [flags]\n  project [command]\n\nAvailable Commands:\n  completion  Generate the autocompletion script for the specified shell\n  create      Create a project\n  delete      Delete a project\n  edit        Edit a project\n  get         Get a project\n  help        Help about any command\n  list        List all projects\n  toggle      Toggle a project\n\nFlags:\n  -h, --help   help for project\n\nUse \"project [command] --help\" for more information about a command.\n")
+}
+
+func TestGetCommand(t *testing.T) {
+	config.ViperNotSet(t)
+
+	httpmock.Activate()
+	defer httpmock.DeactivateAndReset()
+
+	testProject := models.Project{
+		ID:   "testProjectID",
+		Name: "testProjectName",
+	}
+
+	httpmock.RegisterResponder("GET", utils.Host+"/v1/accounts/"+viper.GetString("account_id")+"/projects/"+testProject.ID,
+		func(req *http.Request) (*http.Response, error) {
+			resp, err := httpmock.NewJsonResponse(200, testProject)
+			if err != nil {
+				return httpmock.NewStringResponse(500, ""), nil
+			}
+			return resp, nil
+		},
+	)
+
+	failOutput, _ := utils.ExecuteCommand(ProjectCmd, "get")
+	assert.Equal(t, "Error: required flag(s) \"id\" not set\nUsage:\n  project get [-i <project-id> | --id=<project-id>] [flags]\n\nFlags:\n  -h, --help        help for get\n  -i, --id string   id of the project you want to display\n\n", failOutput)
```

Comment:
I'm wondering if that is not too much details. If tomorrow after a package update, cobra changes the way it displays command helps, you will need to change all those fail output tests. Maybe something like `assert.Contains(t, "required flag(s) \"id\" not set", failOutput)` would be safer ? What do you think ?